IIPS/iip-52.md

```diff
@@ -169,73 +169,70 @@ At this time, the `_from` would be the BTP address of `xcall`.
-ICON to the T1 network, he needs to pay X ICX, and for the T2 network he needs to pay Y ICX.  That is, the fees
-depend on the destination network address.  Also, a `default` fee has been defined in case of there is no fee
-mapping to the network address.
+If a user wants to make a call from ICON to the T1 network, he needs to pay X ICX, and for the T2 network
+he needs to pay Y ICX.  That is, the fees depend on the destination network address.
 
 The fees are divided into two types, one is for relays and the other is for protocol itself.
 For example, for a destination chain T1, the fees could be relayFee = 0.25 ICX and protocolFee = 0.01 ICX.
-And relayFee goes to relays, protocolFee goes to BTP protocol
-(eventually, [Fee Aggregator SCORE](https://github.com/icon-project/IIPs/blob/master/IIPS/iip-35.md)).
-In this document, we don't address how to deal with these accrued fees for distribution.
-For the fees distribution, we need to introduce another follow-up specification.
+And relayFee goes to relays, protocolFee goes to BTP protocol (eventually, to the Fee Handler).
+In this document, we don't address how to deal with these accrued fees for distribution,
+but just define operational parts like how to get the proper fee amount before sending the call request, etc.
 
-Here are getter and setter methods for the fixed fees handling. DApps that want to make a call to `sendCallMessage`,
-should query the total fixed fees for the destination network via `totalFixedFees` interface, and then enclose the
-appropriate fees in the method call.
+Here are getter and setter methods for the proper fees handling in `xcall`.
+DApps that want to make a call to `sendCallMessage`, should query the total fee amount for the destination
+network via `getFee` interface, and then enclose the appropriate fees in the method call.
+Note that the protocol fee amount can be get/set via `xcall`, but the relay fee would be obtained from BMC
+which manages BTP network connections.
 
 ```java
 /**
- * Gets the fixed fee for the given network address and type.
- * If there is no mapping to the network address, `default` fee is returned.
+ * Gets the fee for delivering a message to the _net.
+ * If the sender is going to provide rollback data, the _rollback param should set as true.
+ * The returned fee is the sum of the protocol fee and the relay fee.
  *
  * @param _net The network address
- * @param _type The fee type ("relay" or "protocol")
- * @return The fee amount in loop
+ * @param _rollback Indicates whether it provides rollback data
+ * @return the sum of the protocol fee and the relay fee
  */
 @External(readonly=true)
-BigInteger fixedFee(String _net, String _type);
+BigInteger getFee(String _net, boolean _rollback);
 
 /**
- * Gets the total fixed fees for the given network address.
- * If there is no mapping to the network address, `default` fee is returned.
+ * Sets the protocol fee amount.
```

Comment:
Can you describe in more detail what a protocol commission is?

Reply:
Actually it was an request from @BennyOptions , see https://github.com/icon-project/btp-product/issues/46.
The accrued protocol fees on each chain could be auctioned in exchange for ICX, and this process is another use case of BTP, then eventually could value up of ICX.
For the auction system, see the discussion https://github.com/icon-project/btp/discussions/65.
You may ask @BennyOptions for further behind rationale about this.